### DISCUSSION TITLE
Is it possible to have Payload's postgresql coexis with my website's existing database

### DISCUSSION THREAD

**Discussion by ringge** [payloadcms/payload#5948] [Q&A]
Hello,
Payload is great and I'm trying to integrate it to my existing website, which already had a postgresql database, so was wondering is it possible to let Payload use my existing database? i.e. having it coexist with my current database?
Appreciate any help if possible, thank you

**Comment by dkelsen** [comment]
I'm currently looking into the same thing. I believe you'd have to use two separate database, as my NextJS app uses a different adapter from Payload.

**Reply by ringge** [reply]
Hey there, I think it does not matter if the nextJS app uses a different adapter, as long as we have some way to assign different prefixes to the tables used by the existing site and PayloadCMS. So I was hoping there's a way to do that. It's troublesome to use two separate databases
